hardware_enablement/dpu-hardware-offload.adoc

```diff
@@ -0,0 +1,71 @@
+:_content-type: ASSEMBLY
+[id="offloading-packet-processing-dpu"]
+= OVN/OVS offloading with OpenShift on NVIDIA BlueField-2 DPUs
+include::_attributes/common-attributes.adoc[]
+:context: dpu-hardware-offload
+
+toc::[]
+
+[NOTE]
+====
+The features described in this document are for Developer Preview purposes and are not supported by Red Hat at this time.
+====
+
+Learn how to accelerate and offload software subsystems using {product-title} and the Data Processing Unit (DPU). DPUs are a class of reprogrammable high-performance processors combined with high-performance network interfaces optimized to perform and accelerate network and storage functions carried out by data center servers. This infrastructure element that may also referred to an Intelligence Processing Unit (IPI) by some hardware vendors takes the form of a server hosted PCIe add-in card containing one or more ASICs, usually anchored around a single powerful System on a chip (SoC) device.
+
+The DPU is a complete compute system with an independent software stack, network identity, and provisioning capabilities. The DPU is fully capable of hosting its own applications using either embedded or orchestrated deployment models.
+
+The unique capabilities of the DPU are disruptive because they allow for key infrastructure functions and their associated software stacks to be completely removed from the host node’s CPU cores and to be relocated onto the DPU.
+
+OVN/OVS flow offload is only supported on {product-version} 4.10. 
```

Comment:
This renders to "Branch Build". Is that what we want?

Reply:
@bn222 this will resolve correctly in final output